pymeasure/instruments/hp/hp3478A.py

```diff
@@ -0,0 +1,428 @@
+# -*- coding: utf-8 -*-
```

Comment:
Maybe remove this line; it doesn't really hurt, but it is not common to have this line in any of the other pymeasure files

Reply:
Will remove this line,